OpenRA.Game/Actor.cs

```diff
@@ -202,6 +222,11 @@ public IEnumerable<T> TraitsImplementing<T>()
 			return World.TraitDict.WithInterface<T>(this);
 		}
 
+		public T TraitInfo<T>()
```

Comment:
This is only used in one place in your tests, and adds a redundant path to query the TraitInfo.  It would keep the separation of responsibilities clearer if you dropped this here and from the interface, and keep TraitInfo querying via the ActorInfo.